native_field/src/lib.rs

```diff
@@ -0,0 +1,254 @@
+#![no_std]
+
+extern crate alloc;
+
+use alloc::vec;
+use alloc::vec::Vec;
+use columns::{NativeFieldCols, COL_MAP, NUM_COLS};
+use core::mem::transmute;
+use valida_bus::{MachineWithGeneralBus, MachineWithRangeBus8};
+use valida_cpu::MachineWithCpuChip;
+use valida_machine::{instructions, Chip, Instruction, Interaction, Machine, Operands, Word};
+use valida_range::MachineWithRangeChip;
+
+use p3_air::VirtualPairCol;
+use p3_field::{Field, PrimeField32};
+use p3_matrix::dense::RowMajorMatrix;
+use p3_maybe_rayon::*;
+
+const ADD_OPCODE: u32 = 13;
```

Comment:
Probably not something to change now, but I think it would be nice to have a crate just for these definitions, partly to have them in one place and so they could be used by a future fast interpreter or what not